docs/conf.py

```diff
@@ -29,25 +30,19 @@
 templates_path = ["_templates"]
 html_show_sourcelink = False
 
+# sphinx_favicon
+favicons = [
+    "favicon-16x16.png",
+    "favicon-32x32.png",
+]
+
 # see https://pydata-sphinx-theme.readthedocs.io
 html_theme_options = {
     # "announcement": "Welcome to the new AiiDA site!",
     "logo": {
         "image_light": "logo-light.svg",
         "image_dark": "logo-dark.svg",
     },
-    "favicons": [
-        {
-            "rel": "icon",
```

Comment:
Where do these specifications go in the PR?

Reply:
Is the commit message clear?
https://github.com/aiidateam/aiida-website/pull/71/commits/b51f31552d33b1fb5317928cfda59ac8b76aadd7

Reply:
We don't have to specify, the are automatically determined, but I can also add them 

> For any attributes that you don’t explicitly set, Sphinx Favicon will infer the values from your provided input. For example: if you don’t provide a type attribute, Sphinx Favicon will infer the type from the file extension. If you don’t provide a sizes attribute, Sphinx Favicon will attempt to read the pixel dimensions of the provided image.
https://sphinx-favicon.readthedocs.io/en/stable/configuration.html#configuration

Reply:
I'll just add them back to make it easier